### PR TITLE
Update ESmy006.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Reworked. Quite complex book.

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
Please check when you have time. Thanks!

File: ES/ESmy006.xml [hunk 1]
```diff
@@ -189,128 +592,248 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                           <desc>va: Donation note.</desc>
-                           <quote xml:lang="gez">
-                              <locus target="#73rb">73rb</locus>: ዝመጽሐፍ፡ ፬ወንጌል፡ ዘራስ፡ ስብሐት፡ ዘወሐቦ፡
-                              ለቅዱስ፡ ዮሐንስ፡ ደብረ፡ ማዕፆ፡ ከመ፡ ይኵኖ፡ መርሐ፡ በመንግስተ፡ ሰማያት፡ ወስመ፡ ክርስትናሁኒ፡ ኪዳነ፡
-                              ማርያም፡ ወስመ፡ እሙ፡ ወለተ፡ ስላሴ፡ ወስመ፡ ብእሲቱ፡ ወለተ፡ ማርያም፡ ወስመ፡ ደቂቁ፡ ገብረ፡ ሕይወት፡
-                              ወልደ፡ ገብርኤል፡ አርአያ፡ ሥላሴ፡ ኦአምላከ፡ ዮሐንስ፡ ዕቀብ፡ ሕይወቶ፡ ወስረይ፡ ኀጢአቶ፡
-                                 ወአግርር[<locus target="#73va">73va</locus>]፡ ፀሮ፡ ታህተ፡ እገሪሁ፡ ለገብርከ፡
+                           <locus from="73rb" to="73va"/>
+                           <desc type="DonationNote"/>
+                           <quote xml:lang="gez">ዝመጽሐፍ፡ ፬ወንጌል፡
```
Comment (eu-genia) [inline ES/ESmy006.xml:639]: not `quote `but `q`

File: ES/ESmy006.xml [hunk 2]
```diff
@@ -189,128 +592,248 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                              <persName ref="PRS13292WaldaGabreel">ወልደ፡ ገብርኤል፡</persName> <persName ref="PRS13293ArayaSellase">
+                            አርአያ፡ ሥላሴ፡</persName> ኦአምላከ፡ ዮሐንስ፡ ዕቀብ፡ ሕይወቶ፡ ወስረይ፡ ኀጢአቶ፡
+                                ወአግርር <pb n="73va"/>፡ ፀሮ፡ ታህተ፡ እገሪሁ፡ ለገብርከ፡
                               ኪዳነ፡ ማርያም፡ ወለኵሎሙ፡ ሰብአ፡ ቤቱ፡ ይዕቀቦሙ፡ እመከራ፡ ሥጋ፡ ወነፍስ፡ ለዝንቱኒ፡ መጽሐፋ፡ ወንጌል፡
-                              ዘሠረቆ፡ ወዘፈሐቆ፡ በስልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን፨ ዝ፬ቱ፡ ወንጌል፡ ዘራስ፡ ስብሐት፡
-                              ዘተሣየጦ፡ በንዋዩ፨ </quote>
+                              ዘሠረቆ፡ ወዘፈሐቆ፡ በስልጣነ፡ ጴጥሮስ፡ ወጳውሎስ፡ ውጉዘ፡ ለይኩን፨ ዝ፬ቱ፡ ወንጌል፡
+                              ዘ<persName ref="PRS8668SebhatA">ራስ፡ ስብሐት፡</persName>
+                              ዘተሣየጦ፡ በንዋዩ፨</quote>
                            <quote xml:lang="en">
```
Comment (eu-genia) [inline ES/ESmy006.xml:652]: `<quote xml:lang="en">` should become `<foreign xml:lang="en">`, nested inside the main "`q`" element (as it is not a separate quote but translation of the quote above)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy006.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy006.xml:652]: done

**Pull request merged**